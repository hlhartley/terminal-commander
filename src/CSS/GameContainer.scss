@import '_variables.scss';

.header {
    background: url(/images/grassbackground.png); 
    background-repeat: no-repeat;
    background-size: 100% 100%;
    height: 465px;
}