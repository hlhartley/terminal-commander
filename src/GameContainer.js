import React from 'react';
import './CSS/main.scss';
import ProgressBar from './ProgressBar';

const GameContainer = (props) => {
    return (
        <div className='main-screen'>
           <h1>TERMINAL COMMANDER</h1>
           <ProgressBar />
        </div>
        )
}
export default GameContainer;