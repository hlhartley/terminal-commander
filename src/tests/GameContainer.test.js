import React from 'react';
import GameContainer from '../GameContainer';
import { shallow } from 'enzyme';

let wrapper;
describe('GameContainer', () => {

  beforeEach(() => {
    wrapper = shallow(
      <GameContainer />
    );
  })

  it('should match the snapshot with all data passed in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  })
})
