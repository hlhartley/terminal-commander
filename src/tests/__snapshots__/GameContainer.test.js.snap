// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameContainer should match the snapshot with all data passed in correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <GameContainer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          TERMINAL COMMANDER
        </h1>,
        <div
          className="cloud x1"
        />,
        <div
          className="cloud x2"
        />,
        <div
          className="cloud x3"
        />,
        <div
          className="cloud x4"
        />,
        <div
          className="cloud x5"
        />,
        <ProgressBar />,
      ],
      "className": "main-screen",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "TERMINAL COMMANDER",
        },
        "ref": null,
        "rendered": "TERMINAL COMMANDER",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cloud x1",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cloud x2",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cloud x3",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cloud x4",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cloud x5",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "score": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            TERMINAL COMMANDER
          </h1>,
          <div
            className="cloud x1"
          />,
          <div
            className="cloud x2"
          />,
          <div
            className="cloud x3"
          />,
          <div
            className="cloud x4"
          />,
          <div
            className="cloud x5"
          />,
          <ProgressBar />,
        ],
        "className": "main-screen",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "TERMINAL COMMANDER",
          },
          "ref": null,
          "rendered": "TERMINAL COMMANDER",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cloud x1",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cloud x2",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cloud x3",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cloud x4",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cloud x5",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "score": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
